news/2018-01-20-beatmap-spotlights-december-2017.md

```diff
@@ -0,0 +1,152 @@
+
+## osu!taiko
+
+[![](/wiki/shared/news/2018-01-20-beatmap-spotlights-december-2017/saifu-kamiasobi.jpg)](https://osu.ppy.sh/beatmapsets/627038)
+
+This month has proven that you don't need to be a known mapper or make a 6* map to be featured in, or even top, the monthly Spotlights, because that’s exactly what [ASPIRIN](https://osu.ppy.sh/users/6487115) has achieved !
+His map of the song "Saifu 'Kamiasobi no Uta'" features varied yet consistent and accurate patterns, a storyboard and minor slider velocity changes. ASPIRIN successfully used the song's varied nature to create a fun experience and a nice addition to osu!taiko's ranked map section.
+
+[![](/wiki/shared/news/2018-01-20-beatmap-spotlights-december-2017/powa-of-da-wildanes.jpg)](https://osu.ppy.sh/beatmapsets/693222)
+
+[MMzz](https://osu.ppy.sh/users/128993) is certainly no stranger to the Spotlights, this month bringing us a high-energy rendition of Camellia's POWA OF DA WILDANES. He manages this aggressive song with ease; his patterns flow comfortably while highlighting everything the tune has to offer, from serene moments to frenzied 300 BPM streams and everything in between. This 8* map presents a fun challenge for anyone who dares to face it head-on!
+
+[![](/wiki/shared/news/2018-01-20-beatmap-spotlights-december-2017/fuego.jpg)](https://osu.ppy.sh/beatmapsets/692383)
+
+Following his last appearance in the September osu!taiko Spotlights, [Nifty](https://osu.ppy.sh/users/4956097) is featured once again, this time with a mapset that can be considered a sequel to Time Traveler: sakuraburst's remix of fuego!
+Like Time Traveler, fuego features an unusually high amount of slider velocity changes which are executed very well, creating a very unique experience. The patterns used are exceptionally well-structured, from kantan to embers oni, with the lower difficulties made fair for newer players. Hopefully you, the player, can enjoy embers oni's slider velocity changes!
+
+## osu!catch
+
+[![](/wiki/shared/news/2018-01-20-beatmap-spotlights-december-2017/cycle-hit.jpg)](https://osu.ppy.sh/beatmapsets/692367)
+
+Having impressed us with his previous works, [Spectator](https://osu.ppy.sh/users/702598), in collaboration with [Ascendance](https://osu.ppy.sh/users/2931883), once again amazed us with his newly ranked map KASAI HARCORES - Cycle Hit. With dense yet fairly player-friendly note placement that perfectly fits its complex rhythm, they managed to express this hardcore song outstandingly, ensuring its victory in this month's Spotlights. Be aware of those frequent and hard hyperdashes along with fast sliders appearing occasionally!
```

Comment:
with their newly, plural

Reply:
Be aware of those frequent and hard hyperdashes along with fast sliders appearing occasionally ->
Be aware of frequent and challenging hyperdashes along with high speed sliders
embellishing for drama is 👍